packages/ui/cypress/tests/network-switch.cy.ts

```diff
@@ -0,0 +1,58 @@
+import { accountDisplay } from '../support/page-objects/components/accountDisplay'
+import { settingsPageWatchAccountUrl } from '../fixtures/landingData'
+import { settingsPage } from '../support/page-objects/settingsPage'
+import { topMenuItems } from '../support/page-objects/topMenuItems'
+import { watchMultisigs } from '../fixtures/watchAccounts/watchMultisigs'
+import convertRococoAddressToKusama from '../utils/convertRococoAddressToKusama'
+
+const {
+  publicKey: multisigPublicKey,
+  address: multisigAddress,
+  pureAddress: multisigPureAddress,
+  name: multisigName
+} = watchMultisigs['multisig-with-pure']
+
+describe('Network can be switched', () => {
+  it('should switch account using menu', () => {
+    cy.visitWithLocalStorage({
+      url: settingsPageWatchAccountUrl,
+      accountNames: { [multisigPublicKey]: multisigName as string },
+      watchedAccounts: [multisigPublicKey]
+    })
+
+    cy.url().should('contain', `network=rococo&address=${multisigPureAddress}`)
+
+    settingsPage.accountContainer().within(() => {
+      accountDisplay.identicon().should('be.visible')
+      accountDisplay.nameLabel().should('contain', multisigName)
+      accountDisplay.addressLabel().contains(multisigAddress.slice(0, 5))
+    })
+
+    topMenuItems.desktopMenu().within(() =>
+      topMenuItems
+        .multiproxySelector()
```

Comment:
We can check that the multiproxySelector exists, but I don't think changing the account brings anything to this test.

Reply:
" Let's make sure there's a Multisig in the MultiProxy dropdown for rococo (checking and it's there)"

If I understood you correctly I'm checking for a multisig selector if the watched multisig exists in the dropdown. Or did you mean to check if a multisig dropdown exists?

Reply:
Yes, I mean the drop-down exist. I'm juts trying to come up with tests that change between one network and another. In this case, the selected account has a multisig on Rococo, none on Kusama, that's what needs to be checked.